tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
```

Comment:
for this and line 7 , maybe a if is nicer to read, up to you
i know some use this format as well

Reply:
This is the bash style pattern, usage of "if" is limited to things you cannot put on one liners like this.